src/navbar.ts

```diff
@@ -44,6 +44,10 @@ export const navbar: Navbar = {
           to: "/docs/getting-started/local-development",
           label: "Local Development",
         },
+        {
+          to: "/docs/self-hosted/oel/quickstart",
+          label: "Ory Enterprise",
+        },
```

Comment:
@piotrmsc @aeneasr is this prominent enough? I could make it a top level item instead of a drop down item
![CleanShot 2025-03-07 at 7  42 31@2x](https://github.com/user-attachments/assets/611fd9f3-8161-44bf-897e-6eb5a339a72a)

Reply:
I think it should go under products and either `Ory Enterprise License` or `Self-Host OEL` or `Self-Host Enterprise License`

Reply:
Or alternatively - put it "Products -> Self Hosting"

And under develop:

- local development
- open source development (now called self-hosting)